addons/base_import/static/src/xml/base_import.xml

```diff
@@ -35,6 +35,22 @@
                             </div>
                         </div>
                 </div>
+                <div class="oe_import_with_file oe_import_xlsx row">
+                    <a href="#" class="oe_import_xlsx_toggle col-sm-12">
```

Comment:
The regular import options are toggled because there's several and they're quite complex. Here given for now there's just the sheet picker, the toggle should probably be removed. *However* the box should be hidden if there's only one sheet, no need to povide a choice between one option.

Reply:
ngr thinks the same, I'm moving the option elsewhere